app/src/main/res/layout/fragment_timetable_item_details_page.xml

```diff
@@ -8,30 +8,22 @@
 	android:clickable="true"
 	android:focusable="true"
 	android:orientation="vertical"
-	android:paddingBottom="12dp">
+	android:padding="12dp">
 
 	<TextView
 		android:id="@+id/title"
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
-		android:layout_marginStart="12dp"
-		android:layout_marginTop="12dp"
-		android:layout_marginEnd="12dp"
-		android:layout_marginBottom="12dp"
-		android:paddingStart="12dp"
-		android:paddingLeft="12dp"
-		android:paddingEnd="12dp"
-		android:paddingRight="12dp"
+		android:layout_margin="12dp"
+		android:drawablePadding="24dp"
+		android:drawableStart="@drawable/all_subject"
 		android:textColor="?android:attr/textColorPrimary"
-		android:textSize="20sp"
-		android:textStyle="bold" />
+		android:gravity="center_vertical"/>
```

Comment:
I'd prefer to have the subject displayed as a title like before, tbh

Reply:
Displayed like that it would look a little bit strange because then there would be two almost identical-looking titles but I agree that it should stand out visually.

Maybe something similar to the profiles page with a small header section?

Reply:
![3](https://user-images.githubusercontent.com/36272047/95118569-d952f900-074a-11eb-98e0-c043175d254b.jpeg)